app/services/policies/policies_list_service.rb

```diff
@@ -10,26 +10,46 @@ class Policies::PoliciesListService
 
   class PoliciesListServiceError < RuntimeError; end
 
-  def self.call(account, builtin: true)
+  def self.apicast_registry_url
+    ThreeScale.config.sandbox_proxy.apicast_registry_url
+  end
+
+  delegate :apicast_registry_url, to: 'self.class'
+
+  def self.call(*args)
```

Comment:
Can one actually pass whatever in args?

Reply:
Yep. That means `#call` requires no argument. It's all forwarded to the initializer.

Reply:
@josemigallas You can pass as many arguments as you want, this is called **splat operators** 😄

Reply:
@josemigallas You can check it in https://medium.freecodecamp.org/rubys-splat-and-double-splat-operators-ceb753329a78 👍

Reply:
What sense does this make? The constructor requires account and perhaps builtin, why the static method does not have the same signature? I could throw a confusing error, no?

Reply:
@josemigallas it behaves the same way as if you call to the initialize directly with the wrong params. It is just more dynamic and delegates to it so if you change it there, you don't have to change it to many places. In Java for example (at least in old versions), you can't just delegate 'whatever you receive' and if you change the method arguments, you have to change the whole change of calls, even if that involves many files, here you just need in that same place and that's it.

Reply:
OK if it's the ruby way